content/css/concepts/typography/terms/font-weight/font-weight.md

```diff
@@ -51,3 +51,4 @@ p {
   font-weight: 600;
 }
```

Comment:
[Suggestion] This example is missing some context here. Consider viewing my feedback on [this line](https://github.com/Codecademy/docs/pull/3044/files#r1302110496) to help you. 
```suggestion
  font-weight: 600;
}
```